addons/mass_mailing/static/src/js/mailing_mailing_view_form_full_width.js

```diff
@@ -46,7 +46,7 @@ export class MassMailingFullWidthViewController extends formView.Controller {
         if (!$iframe.length || !$iframe.contents().length) {
```

Comment:
We don't actually handle the act of full-screening in there. It's just a hook to upate the snippets and scrollbar. The actual full-screening is handled by the wysiwyg-iframe, and we modify some stuff in here that's specific to the actual html field behavior (such as the scrollbar being hidden by default)

For repositioning the snippets it doesn't really matter functionally that we do it 4 times since there's no toggling involved, it's just computing positions.